eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -20,6 +20,9 @@ parameters:
 - name: TargetDocRepoName
   type: string
   default: azure-docs-sdk-python
+- name: SkipPythonVersion
```

Comment:
Do we think we will want to allow multiple versions?

Reply:
I would like to rename this parameter to SkipPythonVersions to convey that it support skipping multiple versions.

Reply:
Ok we will have to be careful about how we delimit multiple version (i.e. "3.75" checking for "3.7").

Reply:
This change is not required since we are using exact check